tests/evaluator/test_text_classification.py

```diff
@@ -0,0 +1,73 @@
+import json
+import os
+import subprocess
+import unittest
+from datetime import datetime
+
+from datasets import load_dataset
+from transformers import AutoModelForSequenceClassification, AutoTokenizer, pipeline
+
+from evaluate import evaluator
+
+
+class TestTextClassificationEvaluator(unittest.TestCase):
```

Comment:
Since we probably will only have one test per task we could wrap them all tasks in this class and don't need many files:

```suggestion
class TestEvaluatorTrainerParity(unittest.TestCase):
```